corehq/ex-submodules/casexml/apps/phone/data_providers/case/clean_owners.py

```diff
@@ -65,6 +67,7 @@ def extend_response(self, response):
 
     def _get_next_case_batch(self):
         ids = pop_ids(self.case_ids_to_sync, chunk_size)
+        self._add_load(len(ids))
```

Comment:
any reason to do this for each batch instead of once at the start `self._add_load(len(self.case_ids_to_sync))`. Same for livequery.

Reply:
This will even out the load over time (these stats are recorded as a rate) if there is significant time between processing each batch. Do you see a disadvantage with doing it this way?

Reply:
no, that makes perfect sense. Thanks for explaining.